dev/source/docs/using-gazebo-simulator-with-sitl.rst

```diff
@@ -0,0 +1,205 @@
+.. _using-gazebo-simulator-with-sitl:
+
+================================
+Using Gazebo Simulator with SITL
+================================
+
+This article explains how to use Gazebo http://gazebosim.org/
+as an external simulator for Copter.
+
+Overview
+========
+
+Gazebo is a well-known and respected robotics simulator. We will be compiling 
+Gazebo from source, because no current release has built-in support for ArduCopter.
+
+.. warning::
+   Gazebo support is still under work on October 2016. The following instructions may be outdated or thing could be broken. In those case, open an issue on ardupilot github or discuss.
```

Comment:
maybe rephrase "Gazebo support is still under development, as of October 2016. If it seems that these instructions are outdated, please open an issue on the ardupilot_wiki github."